File: code-coaches/medium/snap-crackle-and-pop/SnapCrackleAndPop.swift
func getSound(numRiceKrispies: Int) -> String {
    if numRiceKrispies % 3 == 0 {
        return "Pop"
    } else if numRiceKrispies % 2 != 0 {
        return "Snap"
    } else {
        return "Crackle"
    }
}

func main() {
    var numRiceKrispiesList: [Int] = []
    for _ in 0..<6 {
        let num = Int(readLine()!)!
		numRiceKrispiesList.append(num)
    }

    var sounds: [String] = []
    for num in numRiceKrispiesList {
        let sound = getSound(numRiceKrispies: num)
        sounds.append(sound)
    }

    print(sounds.joined(separator: " "))
}

main()

File: code-coaches/medium/snap-crackle-and-pop/SnapCrackleAndPop.cs
using System;
using System.Collections.Generic;

public class SnapCrackleAndPop
{
    static string GetSound(int numRiceKrispies)
    {
        if (numRiceKrispies % 3 == 0)
            return "Pop";
        else if (numRiceKrispies % 2 != 0)
            return "Snap";
        else
            return "Crackle";
    }

    static void Main(string[] args)
    {
        List<int> numRiceKrispiesList = new List<int>();
        for (int i = 0; i < 6; i++)
        {
            int num = int.Parse(Console.ReadLine());
            numRiceKrispiesList.Add(num);
        }

        List<string> sounds = new List<string>();
        foreach (int num in numRiceKrispiesList)
        {
            string sound = GetSound(num);
            sounds.Add(sound);
        }

        Console.WriteLine(string.Join(" ", sounds));
    }
}
